fe/fe-core/src/main/java/org/apache/iceberg/hive/IcebergHiveCatalog.java

```diff
@@ -174,6 +180,51 @@ public Database getDB(String dbName) throws InterruptedException, TException {
         return new Database(CONNECTOR_ID_GENERATOR.getNextId().asInt(), dbName);
     }
 
+    @Override
+    public void createDb(String dbName, Map<String, String> properties) {
+        Database database = new Database(CONNECTOR_ID_GENERATOR.getNextId().asInt(), dbName);
+        properties = properties == null ? new HashMap<>() : properties;
+        for (Map.Entry<String, String> entry : properties.entrySet()) {
+            String key = entry.getKey();
+            String value = entry.getValue();
+            if (key.equalsIgnoreCase(LOCATION_PROPERTY)) {
+                try {
+                    URI uri = new Path(value).toUri();
+                    FileSystem fileSystem = FileSystem.get(uri, new Configuration());
+                    fileSystem.exists(new Path(value));
```

Comment:
check the path exists? why not deal the return value?

Reply:
It's mainly to check if the path could access in filesystem